docs/topics/i18n/translation.txt

```diff
@@ -1373,10 +1373,16 @@ function. Example::
 
 .. warning::
 
-    :func:`~django.conf.urls.i18n.i18n_patterns` is only allowed in your root
+    :func:`~django.conf.urls.i18n.i18n_patterns` is only allowed in a root
     URLconf. Using it within an included URLconf will throw an
     :exc:`~django.core.exceptions.ImproperlyConfigured` exception.
 
+.. versionchanged:: 1.10
+
+    Using ``i18n_patterns`` in a root URLconf different from
```

Comment:
Okay, the only other suggestion would be to first describe the feature more than the implementation in the commit message, something like "Allowed using i18n_patterns() in any root URLconf."